### ISSUE TITLE
Failed with error of missing value

### ISSUE THREAD

**Issue by den-ree** [LucianoPAlmeida/bitrise-step-swift-environment-variable-injector#2]
Hi @LucianoPAlmeida  
I'm trying to use your step on bitrise, but there is an error on the last version (0.3.3).

```
Error: Missing value for '--ignore <ignore>'
Usage: variable-injector-tool [--verbose] [--file <file> ...] [--ignore <ignore> ...]
  See 'variable-injector-tool --help' for more information.
```

If I set any value to the ignore field then everything works fine.

**Comment by LucianoPAlmeida** [comment]
Thanks for reporting this!
Yeah, this is being happening since we moved to `swift-argument-parser` because it validates the params. 
There are a couple of solutions for this, but TBH I didn't took the time to do it. I'll try look into this ASAP :)

**Comment by den-ree** [comment]
@LucianoPAlmeida got it, anyway thank you for the solution.
It would be great if you fix it.

**Comment by LucianoPAlmeida** [comment]
018a8b49102f8101ed5556c39fcd2530237417f0 fixes it, I just need to remember how to publish a new step version on bitrise.

**Comment by LucianoPAlmeida** [comment]
I'm going to let this open util the new version is out =]

**Comment by ochalet-wp** [comment]
Thanks for the fix! Do you know when you will be able to publish the new version of this step?

**Comment by LucianoPAlmeida** [comment]
I'll get a shot at this soon, I think is just a matter of remembering the steps required to update it on bitrise ... :)

**Comment by DamienBitrise** [comment]
Hi @LucianoPAlmeida 

You can submit an updated version PR here: https://github.com/bitrise-io/bitrise-steplib

Similar to your previous PR here: https://github.com/bitrise-io/bitrise-steplib/pull/1967

**Comment by LucianoPAlmeida** [comment]
This also should be fixed in 0.1.2
